llvm/test/MachineVerifier/fix-illegal-copies.mir

```diff
@@ -0,0 +1,60 @@
+# RUN: not --crash llc -march=amdgcn -mcpu=gfx1200 -run-pass=none -o /dev/null %s 2>&1 | FileCheck %s    
+# REQUIRES: amdgpu-registered-target    
```

Comment:
Test file name doesn't indicate it's an AMDGPU test. We probably should have an AMDGPU directory at this point too

Reply:
Implemented this in new PR and updated in latest commit.